### ISSUE TITLE
Update or remove e2e test from website

### ISSUE THREAD

**Issue by asklar** [microsoft/react-native-windows-samples#453]
I just realized that we do have an e2e-test page in current, but it's not linked from anything and the info in there is probably stale
We need to either update it, or remove it and link to the [e2e testing .md file ](https://github.com/microsoft/react-native-windows/blob/main/docs/e2e-testing.md) in the repo instead?

[Author and Run E2E Test for React Native Windows · React Native for Windows + macOS (microsoft.github.io)](https://microsoft.github.io/react-native-windows/docs/e2e-test)

**Comment by rectified95** [comment]
Good call. The current correct guidance is the wiki page of the RNW repo (also with a level of indirection, could be simplified) - https://github.com/microsoft/react-native-windows/wiki/E2E-Test
We could first move this content to the website, and also incorporate updated examples of writing tests (I think it mostly focuses of configuring/running them).

**Comment by chrisglein** [comment]
Clarity around how to do UI testing is a common ask. If there's way we can clean this up and leverage that documentation for people outside the framework that'd be good.

**Comment by rectified95** [comment]
@NickGerleman's recent PR https://github.com/microsoft/react-native-windows/pull/8539 moves us closer to external re-usability of our E2E infra. It contains up-to-date descriptions of how use what we currently have. We should use that content for our docs .
